tests/app/main/test_briefs.py

```diff
@@ -527,6 +527,17 @@ def test_non_first_question_shows_previous_page_link(self):
         assert (doc.xpath("//a[text()='Back to previous page']/@href")[0] ==
                 '/suppliers/opportunities/1234/responses/5/availability')
 
+    def test_respond_to_email_previous_page_link_skips_nice_to_have_requirements_if_no_nice_to_haves_for_brief(self):
+        self.brief['briefs']['niceToHaveRequirements'] = []
```

Comment:
Have checked manually and it is currently being saved as an empty list when creating a brief. I'll add to the tech debt board